docker-orchestration/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-buster
+
+RUN apt-get update \
+    && apt-get -q -y install git gcc nginx python3-libvirt libxslt1-dev libxml2-dev qemu-kvm libvirt-dev virt-manager mediainfo
```

Comment:
qemu-kvm, libvirt, virt-manager, nginx won't be needed anymore in the new setup

Reply:
fixed via 32fb56d